evap/staff/views.py

```diff
@@ -1105,27 +1150,24 @@ def user_import(request):
         if operation not in ('test', 'import'):
             raise SuspiciousOperation("Invalid POST operation")
 
-        test_run = operation == 'test'
-        import_run = operation == 'import'
-
-        if test_run:
-            delete_import_file(request.user.id)  # remove old file if still exists
+        if operation == 'test':
+            delete_import_file(request.user.id, magic)  # remove old files if still exist
             excel_form.excel_file_required = True
             if excel_form.is_valid():
                 excel_file = excel_form.cleaned_data['excel_file']
                 file_content = excel_file.read()
-                __, success_messages, warnings, errors = UserImporter.process(file_content, test_run)
+                __, success_messages, warnings, errors = UserImporter.process(file_content, True)
                 if not errors:
-                    save_import_file(excel_file, request.user.id)
+                    save_import_file(excel_file, request.user.id, magic)
 
-        elif import_run:
-            file_content = get_import_file_content_or_raise(request.user.id)
-            __, success_messages, warnings, __ = UserImporter.process(file_content, test_run)
+        elif operation == 'import':
+            file_content = get_import_file_content_or_raise(request.user.id, magic)
+            __, success_messages, warnings, __ = UserImporter.process(file_content, False)
```

Comment:
i think i would actually name that parameter, `test_run=False`, here and in the other views. the views are complex enough without looking up parameter names.